skbuild/resources/cmake/UseF2PY.cmake

```diff
@@ -0,0 +1,149 @@
+#.rst:
+#
+# The following functions are defined:
+#
+# .. cmake:command:: add_f2py_target
+#
+# Create a custom rule to generate the source code for a Python extension module
+# using f2py.
+#
+#   add_f2py_target(<Name> [<F2PYInput>]
+#                   [OUTPUT_VAR <OutputVar>])
+#
+# ``<Name>`` is the name of the new target, and ``<F2PYInput>``
+# is the path to a cython source file.  Note that, despite the name, no new
+# targets are created by this function.  Instead, see ``OUTPUT_VAR`` for
+# retrieving the path to the generated source for subsequent targets.
+#
+# If only ``<Name>`` is provided, and it ends in the ".pyf" extension, then it
+# is assumed to be the ``<F2PYInput>``.  The name of the input without the
+# extension is used as the target name.  If only ``<Name>`` is provided, and it
+# does not end in the ".pyf" extension, then the ``<F2PYInput>`` is assumed to
+# be ``<Name>.pyf``.
+#
+# The Cython include search path is amended with any entries found in the
```

Comment:
This paragraph could probably be removed as include directories are not used. See other comment below.